templates/zabbix-check-fmg-cert.sh.j2

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# called by Zabbix to report days-left on the Fedora Messaging cert
+
+PATH=$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
+
+# Get the expiry date and convert to epoch seconds
+
+expiry_date=$(openssl x509 -text -in "/etc/fedora-messaging/{{ fedora_messaging_tls_cert }}" | grep "Not After" | sed 's/.*Not After : //')
+expiry_epoch=$(date -d "$expiry_date" +%s)
+
+# Get current time in epoch seconds
+current_epoch=$(date +%s)
+
+# Calculate days left
+days_left=$(( (expiry_epoch - current_epoch) / 86400 ))
+
+echo $days_left
```

Comment:
that's just sending $days_left to STDOUT but not to zabbix ? Forgot to add zabbix_sender call ?

Reply:
Done